runelite-client/src/main/java/net/runelite/client/plugins/timersandbuffs/TimersAndBuffsPlugin.java

```diff
@@ -477,7 +476,8 @@ else if (nextSuperAntifireTick - tickCount <= 0)
 		{
 			if (client.getVarbitValue(Varbits.DIVINE_SUPER_COMBAT) > event.getValue()
 				|| client.getVarbitValue(Varbits.DIVINE_BASTION) > event.getValue()
-				|| client.getVarbitValue(Varbits.DIVINE_BATTLEMAGE) > event.getValue())
+				|| client.getVarbitValue(Varbits.DIVINE_BATTLEMAGE) > event.getValue()
+				|| client.getVarbitValue(Varbits.MOONLIGHT_POTION) > event.getValue())
```

Comment:
Also, similarly to how drinking a moonlight potion removes existing divine super defence potion infoboxes, drinking a divine super defence potion (or having this var update from any of the other divine potions which provide its effect) should remove the moonlight potion's infobox, since it preserves the defence boost beyond the 5 minutes provided by the moonlight potion.

That would should be added after this guard clause, eg.

```java
if (client.getVarbitValue(Varbits.MOONLIGHT_POTION) < event.getValue())
{
	removeVarTimer(MOONLIGHT_POTION);
}
```

Reply:
Regarding https://github.com/runelite/runelite/pull/17662#discussion_r1677028493:
With ``one tick`` you are referring to how the divine super def seems to send the varbitchanged event before the moonlight potion? I.e. on that gametick after the super def var value has --'d, before the moonlight potion var value has --'d, super def var == moonlight potion var?
That'd leave the moonlight potion timer 1 gametick incorrect though, no? What do you think of changing it to ``>=`` as you suggested, and just removing:
```java
if (client.getVarbitValue(Varbits.DIVINE_SUPER_DEFENCE) == moonlightValue)
{
	removeVarTimer(DIVINE_SUPER_DEFENCE);
}
```
(i.e. not completely removing the +1 hack, just the part that removes the super def timer every tick).
I've implemented this change for now.

Reply:
That's fine, however the comment in the `MOONLIGHT_POTION` block isn't really very useful now; it would be helpful to explain that without that code, the timer would be 1 tick shorter than it is meant to be.